Automattic/sensei
# Fix settings form not redirecting to the correct tab when submitted

Resolves https://github.com/Automattic/sensei/issues/7351
Related to #7415

## Proposed Changes

This PR fixes the settings form always redirecting to the first (General) tab when the form is saved.

It also fixes the active tab highlight not working on the tabs menu.

It changes the URL structure to using params instead of hash (`#default-settings` -> `&tab=default-settings`) but the old structure is still supported.

## Testing Instructions


1. Go to Sensei LMS -> Settings -> Appearance (you could choose any tab that uses a hash in the URL except the first one)
2. Click the "Save Changes" button.
3. You should be redirected to the same tab and not to the first one (General).
4. Check if the active tab is highlighted in the tabs menu.
5. Check if the `Emails` tab is working as before.
6. Check if the old hash URL structure is still working.

## Pre-Merge Checklist
